drivers/scheduler/k8s/k8s.go

```diff
@@ -6346,13 +6346,26 @@ func getLabelsFromNodeAffinity(nodeAffSpec *v1.NodeAffinity) map[string]string {
 	return label
 }
 
+// mergeMaps merges two maps
+func mergeMaps(m1 map[string]string, m2 map[string]string) map[string]string {
```

Comment:
How about

`func mergeMaps[K comparable, V any](m1 map[K]V, m2 map[K]V) map[K]V {
	merged := make(map[K]V)
	for k, v := range m1 {
		merged[k] = v
	}
	for k, v := range m2 {
		merged[k] = v
	}
	return merged
}
`

Reply:
Done

Reply:
Might be useful in future.